src/ms3/score.py

```diff
@@ -1217,7 +1241,7 @@ def store_excerpt(
         excerpt.store_score(excerpt_filepath)
         self.logger.info(f"Excerpt for MCs {start}-{end} stored at {excerpt_filepath}.")
 
-    def store_phrase_excerpts(self, directory: Optional[str] = None):
+    def find_phrases(self, directory: Optional[str] = None):
         """Store excerpts based on the phrase annotations contained in the score, if any. For this purpose,
```

Comment:
please adapt docstring accordingly and annotate output type. 

AFAICT, the `directory` parameter is not used in this method and I think it would be better called something like `get_phrase_boundaries`?

Reply:
Sorry about this one, I didn't see that the docstring was completely wrong :)